WordPress/wordpress-playground
# Ship the OpenSSL extension with the kitchen sink version of PHP.wasm

## What is this PR doing?

Related to https://github.com/WordPress/wordpress-playground/issues/655

Includes the OpenSSL PHP extension in the `kitchen-sink` extension bundle. This also makes it available via the "Load extensions" checkbox on playground.wordpress.net.

## Testing instructions

1. Apply this PR
2. Open Playground and go to /phpinfo.php
3. Confirm the OpenSSL extension is listed as available
